src/main/java/io/github/antishake/AntiShake.java

```diff
@@ -41,35 +39,37 @@
 
   // To load the config.properties when the class is loaded
   static {
-    InputStream is = null;
-    try {
-      properties = new Properties();
-      is = ClassLoader.class.getResourceAsStream("/config.properties");
-      properties.load(is);
-      loadProperties();
-    } catch (FileNotFoundException e) {
-      logger.error(e);
-    } catch (IOException e) {
-      logger.error(e);
-    }
```

Comment:
Oh I see that inside the constructor.. It should be fine for now as the library is accessed by only one application. But, we may need to load it as soon as the library is loaded if we have more applications accessing it..